tests/test_wave_07.py

```diff
@@ -0,0 +1,52 @@
+import pytest
```

Comment:
Great job setting up tests for the enhancement wave!! Another test you can look out for is handling ties for newest age!